--- mutagen-1.42.0/mutagen/apev2.py~	2018-11-17 10:56:49.000000000 +0000
+++ mutagen-1.42.0/mutagen/apev2.py	2022-08-19 21:46:55.162926190 +0000
@@ -32,7 +32,10 @@
 
 import sys
 import struct
-from collections import MutableSequence
+if sys.version_info[:2] >= (3, 8):
+    from collections.abc import MutableSequence
+else:
+    from collections import MutableSequence
 
 from ._compat import (cBytesIO, PY3, text_type, PY2, reraise, swap_to_string,
                       xrange)
